jwst/fits_generator/proposalparser.py

```diff
@@ -49,7 +49,7 @@ def get_detectors(filename):
                         ExposureNum = 0
                         for exposure in activity:
                             ExposureNum = ExposureNum + 1
-                            DetectorNum = 0
+                            DetectorNum = 0 # noqa F841
```

Comment:
same as above

Reply:
From the git history, it doesn't look like that value has ever been used.  It can probably be removed.

Reply:
```suggestion
```